packages/Input/src/view/Text/Input.tsx

```diff
@@ -6,6 +6,11 @@ import { LoadingIcon } from './Icons/LoadingIcon';
 import { SuccessIcon } from './Icons/SuccessIcon';
 import { WarningIcon } from './Icons/WarningIcon';
 
+
```

Comment:
Can you also add to [this file](https://github.com/thebyte9/blaze-components-react/blob/f4237e20d73c21528cc679e99480a2c9e793ced6/packages/Input/src/deprecated/Input.tsx)